src/serialmonitor/serialMonitor.ts

```diff
@@ -206,6 +226,13 @@ export class SerialMonitor implements vscode.Disposable {
         this._baudRateStatusBar.text = chosen;
     }
 
+    public async changeTimestampFormat() {
+        const timestampFormat = await vscode.window.showInputBox();
+        await this._serialPortCtrl.changeTimestampFormat(timestampFormat);
+        this._currentTimestampFormat = timestampFormat;
+        this._timestampFormatStatusBar.text = timestampFormat;

```

Comment:
Similar to the previous comment, it might be better to keep the text of the status bar more readable. If you want to show the current format, maybe it could go in the tooltip.

Reply:
You should consider adding a check to ensure the user gave a valid timestamp.

Reply:
> Similar to the previous comment, it might be better to keep the text of the status bar more readable. If you want to show the current format, maybe it could go in the tooltip.

Changed in https://github.com/microsoft/vscode-arduino/pull/1430/commits/0628fb09e3a44ed530a4df8d45c9bb25eeab6221

Reply:
> You should consider adding a check to ensure the user gave a valid timestamp.

What do you mean by valid timestamp? It's up to user what do they insert there...
But, I guess, that you meant that `timestampFormat` should contain `%`.